### PR TITLE
OMI-8 / OMI-12 Generate Random ID on insuree Creation and add Insuree attachment

### PR DESCRIPTION
**Pull request by mngoe**
Random ID génération on Insuree Creation process

### CODE REVIEW

File: insuree/migrations/0014_insureeattachment.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='InsureeAttachment',
+            fields=[
+                ('idAttachment', models.AutoField(primary_key=True, serialize=False)),
+                ('folder', models.CharField(db_column='Folder', max_length=250)),
+                ('name', models.CharField(db_column='FileName', max_length=250)),
+                ('insuree', models.ManyToManyField(to='insuree.Insuree')),
```
Comment (dragos-dobre) [inline insuree/migrations/0014_insureeattachment.py:19]: Is there a case when an attachment is attached to multiple insurees?

File: insuree/models.py [hunk 2]
```diff
@@ -298,6 +298,23 @@ class Meta:
...
+class InsureeAttachment(models.Model):
+    """ Class Attachment :
+    Class for isurees attachments
+    """
+    idAttachment = models.AutoField(
+        primary_key=True
+    )
+    folder = models.CharField(db_column='Folder', max_length=250, null=False)
+    name = models.CharField(db_column='FileName', max_length=250, null=False)
+    insuree = models.ManyToManyField('Insuree')
```
Comment (dragos-dobre) [inline insuree/models.py:310]: Is there a case when an attachment is attached to multiple insurees?

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

**Comment by dragos-dobre** [comment]
@mngoe Did you started from develop branch?

File: insuree/models.py (hunk 2 above)
Comment (mngoe) [reply insuree/models.py]: Fix to change cardinality Pushed

**Comment by mngoe** [comment]
> @mngoe Did you started from develop branch?

@dragos-dobre As discussed, Develop branch can be unstable and difficult to customize (as a lot of fixes are done on multiple repository and update from one repository can break other part of the code; we experienced a lot of difficulties during Cheque Santé project to manage such a volatile development environment)

For such a short development (our customs could be finished next week), I prefered to begin work on main-branch.

**Review by delcroip — APPROVED** [review]

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**